membership_file/templates/membership_file/no_member.html

```diff
@@ -7,8 +7,9 @@
   Squire - Not a Member
 {% endblock title %}
 
+
+
```

Comment:
NIT: why add empty lines?

Reply:
That probably happened accidentally when I was changing all templates that extend core/base.html to no longer contain the contentFrame attribute and I accidentally allocated the space for the frame processing.